packages/components/Button/src/Button.tsx

```diff
@@ -64,19 +66,25 @@ export const Button = compose<IButtonType>({
       },
       content: { children: content, testID: testID },
       icon: createIconProps(icon),
+      chevron: {},
     });
 
     return { slotProps, state };
   },
   settings,
   render: (Slots: ISlots<IButtonSlotProps>, renderData: IButtonRenderData, ...children: React.ReactNode[]) => {
     const info = renderData.state!.info;
+    const xml = `
+    <svg width="11" height="6" viewBox="0 0 11 6">
+      <path fill='currentColor' d="M0.646447 0.646447C0.841709 0.451184 1.15829 0.451184 1.35355 0.646447L5.5 4.79289L9.64645 0.646447C9.84171 0.451185 10.1583 0.451185 10.3536 0.646447C10.5488 0.841709 10.5488 1.15829 10.3536 1.35355L5.85355 5.85355C5.65829 6.04882 5.34171 6.04882 5.14645 5.85355L0.646447 1.35355C0.451184 1.15829 0.451184 0.841709 0.646447 0.646447Z" />
+    </svg>`;
```

Comment:
Where did the SVG come form? That feels like a complicated path for what I assume is just an arrow?

Reply:
I had to do a similar thing to my submenu chevron. The svg is a chevron from our Office icon directory, and since we don't have a way of importing it directly, we have to implement it as an inline svg.

Reply:
Ah OK. Inline SVG makes sense, I was more commenting on the actual path / numbers inside the SVG.

I've noticed that SVGs that came from Figma seem to be more complicated than need be. We use an SVG for ActivityIndicator, and also create the SVG inline. Despite the visuals looking the same, the svg that we would export from figma is a lot more complicated (a lot more points on the path to define the arc). Not a blocker just a thing I noticed.

Reply:
I got an icon from Figma file. I saved it as SVG